### ISSUE TITLE
is the HT1080Z working?

### ISSUE THREAD

**Issue by rampa069** [mist-devel/mist-board#135]
Hi. i think he HT1080Z is not working. i have the rom in the SD root folder but after starting it  i only get "@" in the screen. (Also resintetized but same result)

also tried the rbf from http://joco.homeserver.hu/fpga/mist_ht1080z_en.html with the same result :-(

**Comment by gyurco** [comment]
I have no idea, it works for me.

**Comment by rampa069** [comment]
and the rom place is in the root folder, isnt it?

**Comment by gyurco** [comment]
Yes, if you don't have any rom= lines in your mist.ini. Fsck your SD Card.

**Comment by harbaum** [comment]
That would also be my recommendation. Use a freshly formatted sd card and put only the core (renamed to core.rbf) and the ROM into the cards root directory. Nothing else.

**Comment by rampa069** [comment]
tried all. The problem i think are the new sdram timings. something similar happened to SNES core. Is working with older SDRAM chips.

**Comment by gyurco** [comment]
The SDRAM is clocked only @42Mhz, and the controller is not particularly different than in any other 8 bit core. Are Colecovision/C16/C64/ZX81...etc are working?
SNES is a very different beast, running the SDRAM @128Mhz, without any NOP cycles.

**Comment by gyurco** [comment]
Ok, maybe this one:
change the "set_locaton_assignment PLL_1 ...." in the .qsf to:
````
set_location_assignment PLL_1 -to clkmgr|altpll_component|auto_generated|pll1
